Added an offering; needs a Flexbooker link.

jbrains opened the pull request:
Thanks!

jbrains committed:
Added an offering; needs a Flexbooker link.

jlangr commented:
thanks JB. Merged and added a flexbooker link.

Take care with scheduling. It looks like when you enter a time using the instructions I sent, it is in my time (US Mountain); I've asked for them to fix this. Make sure you set your calendar appropriately, look at what's scheduled, and adjust appropriately.

Flexbooker looks a little green but so far they are pretty responsive. I suspect their original model was for physical locations in a single time zone.

jbrains commented:
Fortunately I noticed the time zone issue and I believe I have adjusted
accordingly. Thank you for checking.

J. B. Rainsberger :: tdd.training :: www.jbrains.ca

Den lör 8 aug. 2020 21:57Jeff Langr <notifications@github.com> skrev:

> thanks JB. Merged and added a flexbooker link.
>
> Take care with scheduling. It looks like when you enter a time using the
> instructions I sent, it is in my time (US Mountain); I've asked for them to
> fix this. Make sure you set your calendar appropriately, look at what's
> scheduled, and adjust appropriately.
>
> Flexbooker looks a little green but so far they are pretty responsive. I
> suspect their original model was for physical locations in a single time
> zone.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/pubmob-com/pubmob/pull/6#issuecomment-670990377>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAPAQM3CQTMTOXHGKWFMRTR7XX5XANCNFSM4PYZW6HA>
> .
>
